test/1_stdlib/UnsafePointer.swift.gyb

```diff
@@ -350,47 +396,16 @@ UnsafePointerTestSuite.test("customMirror") {
 #endif
 }
 
-UnsafePointerTestSuite.test("customPlaygroundQuickLook") {
+${SelfName}TestSuite.test("customPlaygroundQuickLook") {
   // Ensure that the custom playground quicklook works properly, including when
   // the raw value is greater than Int.max
   let reallyBigInt: UInt = UInt(Int.max) + 1
-  let ptr = UnsafePointer<Float>(bitPattern: reallyBigInt)!
-  if case let .text(desc) = ptr.customPlaygroundQuickLook {
-#if arch(i386) || arch(arm)
-    expectEqual("UnsafePointer(0xFFFFFFFF80000000)", desc)
-#elseif arch(x86_64) || arch(arm64) || arch(powerpc64) || arch(powerpc64le) || arch(s390x)
-    expectEqual("UnsafePointer(0x8000000000000000)", desc)
-#else
-    fatalError("Unimplemented")
-#endif
-  } else {
-    expectTrue(false)
-  }
-}
-
-UnsafeMutablePointerTestSuite.test("customMirror") {
-  let reallyBigInt: UInt = UInt(Int.max) + 1
-  let ptr = UnsafeMutablePointer<Float>(bitPattern: reallyBigInt)!
-  let mirror = ptr.customMirror
-  expectEqual(1, mirror.children.count)
-#if arch(i386) || arch(arm)
-  expectEqual("18446744071562067968", String(mirror.children.first!.1))
-#elseif arch(x86_64) || arch(arm64) || arch(powerpc64) || arch(powerpc64le) || arch(s390x)
-  expectEqual("9223372036854775808", String(mirror.children.first!.1))
-#else
-  fatalError("Unimplemented")
-#endif
-}
-
-UnsafeMutablePointerTestSuite.test("customPlaygroundQuickLook") {
-  let reallyBigInt: UInt = UInt(Int.max) + 1
-  let ptr = UnsafeMutablePointer<Float>(bitPattern: reallyBigInt)!
-  let isProperDisposition : Bool
+  let ptr = ${SelfType}(bitPattern: reallyBigInt)!
   if case let .text(desc) = ptr.customPlaygroundQuickLook {
 #if arch(i386) || arch(arm)
```

Comment:
No tests got deleted. Previously the tests were copy-pasted for no reason. Now one copy of the tests works for all pointer types, including raw pointers.